lib/cisco_node_utils/cmd_ref/bgp_neighbor.yaml

```diff
@@ -8,7 +8,7 @@ _template:
   context:
     - "router bgp <asnum>"
     - "(?)vrf <vrf>"
-    - "neighbor <nbr>"
+    - "(?)neighbor <nbr>"
```

Comment:
Not in favor of this change  neighbor is *generally* mandatory. There are only a few specific cases in this file where it is optional (`all_neighbors` is the only one I can think of offhand) and in those cases they explicitly redefine their context. For every other property in this file, it's mandatory and should be kept as such.

Same comment for the next file as well.